cmflib/commands/init/show.py

```diff
@@ -0,0 +1,59 @@
+###
+# Copyright (2022) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+#!/usr/bin/env python3
+import argparse
+import os
+import subprocess
+import shlex
+
+from cmflib import cmfquery
+from cmflib.cli.command import CmdBase
+from cmflib.cli.utils import read_cmf_config, find_root
+
+
+class CmdInitShow(CmdBase):
+    def run(self):
+        msg = "'cmf' is not configured.\nExecute 'cmf init' command."
+        result = subprocess.run(
+            ["dvc", "config", "-l"],
```

Comment:
Handline of subprocess run should be made robust everywhere

Reply:
Done